ssv/proposer.go

```diff
@@ -124,12 +122,7 @@ func (r *ProposerRunner) ProcessConsensus(signedMsg *qbft.SignedMessage) error {
 	}
 
 	// specific duty sig
-	var blkToSign ssz.HashRoot
-	if r.decidedBlindedBlock() {
-		blkToSign, err = decidedValue.GetBellatrixBlindedBlockData()
-	} else {
-		blkToSign, err = decidedValue.GetBellatrixBlockData()
-	}
+	blkToSign, err := decidedValue.GetBlockRoot()
```

Comment:
should probably be in this file not consensus data

Reply:
done